ui/src/app/pages/apps-routes/app-logs/app-logs.page.ts

```diff
@@ -11,8 +11,11 @@ import { ErrorToastService } from 'src/app/services/error-toast.service'
 })
 export class AppLogsPage {
   @ViewChild(IonContent, { static: false }) private content: IonContent
+  page = 1
   pkgId: string
-  logs = ''
+  firstTimeLoaded = false
+  needInfinite = true
```

Comment:
Will initializing `needInfinite` to true cause an issue if it's not actually needed?